.nais/naiserator.yaml

```diff
@@ -36,5 +36,9 @@ spec:
     requests:
       cpu: 50m
       memory: 128Mi
+  inbound:
+    rules:
+      - application: syfooppfolgingsplanservice
+        namespace: team-esyfo
```

Comment:
Jeg har det for meg at man kanskje må ha med cluster også når de kjører i forskjellige cluster?

Reply:
Eller har du testet at det fungerer i dev?

Reply:
@kaja-nav Nei, hvis man utelatter cluster så fungerer tar den inn traffic fra både fss og gcp. Tenkte det er likegreit å ha den for begge til vi flytter syfooppfolgingsplanservice til gcp. Har testet at det fungerte i dev med ved å opprette oppfolgingsplaner og se at pdf ble riktig generert.

Reply:
@kaja-nav Ja, det har du visst rett i! Jeg har tester i dev ved å opprette oppfolgingsplan og sjekke at pdf ble generert riktig 👍

Reply:
Ja, det har du rett i 👍

Reply:
Tester nå at pdf-genering fungerte ved å slette poden i fss, og sjekke at PDF fortsatt ble generert rikitg ✔️